assignments/function-conversion.js

```diff
@@ -1,21 +1,21 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
+// let myFunction = () => {
```

Comment:
`let`, `const`, `var` allows us to create variable binding names, `let` and `var` are both mutable, meaning we can change them. most of the time though we don't want to have that effect especially when declaring functions. we would use `const` for most of the time when we declare these bindings. and if the variable does need to change over time then we would consider using `let`, but we should always default to `const`